js/apps/system/_admin/aardvark/APP/frontend/js/views/documentsView.js

```diff
@@ -671,6 +671,20 @@
               if (!this.collection.getIsEnterpriseEdge()) {
                 // We do show the _key globally for every type of edge collection,
                 // except in case we do have an "EnterpriseSmart" edge.
+                var documentNameValidations = [
+                  {
+                    rule: Joi.string().regex(/^[a-zA-Z0-9_\-:\.@()\+,=;$!*\%'%]+$/),
+                    msg: "Only these characters are allowed: a-z, A-Z, 0-9 and  _ - : . @ ( ) + , = ; $ ! * ' %."
+                  },
+                  {
+                    rule: Joi.string().max(254, 'utf8'),
+                    msg: 'Document key max length is 254.'
+                  },
+                  {
+                    rule: Joi.string().allow('').optional(),
```

Comment:
Not sure what this rule means, but an empty document key is not allowed.

Reply:
they key is omitted, this was already the case, no change made there